test/testProxMaps.jl

```diff
@@ -158,7 +158,7 @@ function testNuclear(N=32,rank=2;σ=0.05)
   @test 0.5*norm(xNoisy-x_lr)^2+normNuclear(x_lr,5*σ,svtShape=(N,N)) <= normNuclear(xNoisy,5*σ,svtShape=(N,N))
 end
 
-function testLLR(shape=(32,32,80),blockSize=[4,4];σ=0.05)
+function testLLR(shape=(32,32,80),blockSize=(4,4);σ=0.05)
```

Comment:
Is it possible to showcase by an additional test that your implementation is a generalization?

Reply:
Absolutely! I added a simple test for 3D. It is incomplete, though, as the norm itself is not yet implemented in ND. I can do so some day, but I am a bit swamped right now.